From: Andrea Azzarone <andrea.azzarone@canonical.com>
Date: Mon, 13 Aug 2018 17:38:57 +0000
Subject: goa: Add Ubuntu Single Sign-On to the list

Show "Ubuntu Single Sign-on" first in the list in online accounts page.

Forwared: not-needed
---
 gnome-initial-setup/pages/goa/gis-goa-page.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnome-initial-setup/pages/goa/gis-goa-page.c b/gnome-initial-setup/pages/goa/gis-goa-page.c
index 87341c4..bc135fc 100644
--- a/gnome-initial-setup/pages/goa/gis-goa-page.c
+++ b/gnome-initial-setup/pages/goa/gis-goa-page.c
@@ -186,7 +186,7 @@ populate_provider_list (GisGoaPage *page)
   g_auto(GStrv) conf_providers =
     gis_driver_conf_get_string_list (GIS_PAGE (page)->driver, VENDOR_GOA_GROUP, VENDOR_PROVIDERS_KEY, NULL);
   GStrv providers = conf_providers ? conf_providers :
-    (gchar *[]) { "google", "owncloud", "windows_live", "facebook", NULL };
+    (gchar *[]) { "ubuntusso", "google", "owncloud", "windows_live", "facebook", NULL };
 
   /* This code will read the keyfile containing vendor customization options and
    * look for options under the "goa" group, and supports the following keys:
